missilewars-plugin/src/main/java/de/butzlabben/missilewars/listener/GameListener.java

```diff
@@ -90,87 +89,59 @@ public void onMove(PlayerMoveEvent event) {
     public void onExplode(EntityExplodeEvent event) {
         if (!isInGameWorld(event.getLocation())) return;
 
-        Game game = getGame();
+        if (event.getEntity().getType() != EntityType.FIREBALL) return;
+        if (!getGame().getArena().getFireballConfiguration().isDestroysPortal()) return;
```

Comment:
This if clause is reversed
```suggestion
        if (getGame().getArena().getFireballConfiguration().isDestroysPortal()) return;
```

Reply:
If the "destroy_portal" is `false` RETURN and don't delete the portals in the next step. Make sense for me.

Reply:
The following code will remove the portals from the list of effected blocks. So with your code if `destroys_portal` is true, then the code will progress and remove the portal blocks from the list of effected blocks and the portal won't be destroyed.

Reply:
Oh, right. I misunderstand the target behind the `blockList().removeIf` action.